rdrf/rdrf/models.py

```diff
@@ -2002,6 +2024,21 @@ def get_default_name(self, patient_model):
         else:
             return "Modules"
 
+    def get_value_from_cde(self, patient_model, context_model):
+        form_name, section_code, cde_code = self.naming_cde_to_use.split("/")
+        try:
+            cde_value = patient_model.get_form_value(self.registry.code,
```

Comment:
well, not sure. I did what the `get_name_from_cde` below was doing.
Should these be changed?